src/status.cc

```diff
@@ -27,11 +36,13 @@
 using namespace std;
 
 StatusPrinter::StatusPrinter(const BuildConfig& config)
-    : config_(config),
-      started_edges_(0), finished_edges_(0), total_edges_(0), running_edges_(0),
-      time_millis_(0), progress_status_format_(NULL),
+    : config_(config), started_edges_(0), finished_edges_(0), total_edges_(0),
+      running_edges_(0), time_millis_(0), cpu_time_millis_(0),
+      time_predicted_percentage_(0.0), eta_predictable_edges_total_(0.0),
+      eta_predictable_cpu_time_total_(0.0), eta_predictable_edges_remaining_(0),
+      eta_predictable_cpu_time_remaining_(0),
+      eta_unpredictable_edges_remaining_(0), progress_status_format_(NULL),
```

Comment:
Now that we have C++11: For newly added member variables, lets move their initialization and default value to the same place in the class body.

Continuing to have to maintain this list of initializes is a major eye-sore and prone to mistakes.